### PR TITLE
Project 5: Views.

### PR DESCRIPTION
**Pull request by er-contreras**
- Implement the design from the sneak peek wireframes

Type: `rails db:setup` please

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,16 +1,15 @@
-
-<ul id='users_section'>
+<div id='users_section'>
   <% @users.each do |user| %>
```
Comment (rloterh) [inline app/views/users/index.html.erb:2]: - [ ] OPTIONAL
As a best practice, you may want to opt for `<%= render @users %>` in place of `@users.each`

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -1,16 +1,15 @@
-
-<ul id='users_section'>
+<div id='users_section'>
```
Comment (rloterh) [inline app/views/users/index.html.erb:1]: - [ ] Your application is not running as intended. When your application is launched it displays a blank screen, with no content. Kindly ensure to improve upon this aspect

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Erick,

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Davidosky007** [comment]
## Hi @eri8-9 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
CR GIF



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**